src/collective/upload/profiles/uninstall/jsregistry.xml

```diff
@@ -9,7 +9,7 @@
   <javascript remove="True" id="++resource++collective.upload/jquery.fileupload.js"/>
   <javascript remove="True" id="++resource++collective.upload/jquery.fileupload-fp.js"/>
   <javascript remove="True" id="++resource++collective.upload/jquery.fileupload-ui.js"/>
-  <javascript remove="True" id="++resource++collective.upload/vendor/bootstrap.min.js"/>
+  <javascript remove="True" id="++resource++collective.js.bootstrap/js/bootstrap.min.js"/>
```

Comment:
@rodfersou you must not remove this here as other add-ons may still be using it.